### PR TITLE
feat(services): add Helm support in service pages

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-874

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Write comments that add new information that doesn’t exist in the code

File: libs/domains/services/feature/src/lib/service-details/service-details.tsx [hunk 1]
```diff
@@ -166,71 +255,34 @@ export function ServiceDetails({ className, environmentId, serviceId, ...props }
...
-              serviceType: ServiceTypeEnum.JOB,
+              serviceType: 'JOB',
               source: P.when(isJobGitSource),
             },
+            {
+              serviceType: 'HELM',
+              source: P.when(isHelmGitSource),
+            },
             (service) => {
               const gitRepository = match(service)
```
Comment (RemiBonnet) [inline libs/domains/services/feature/src/lib/service-details/service-details.tsx:268]: It doesn't work we can't see a git repository source for Helm

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Hey @ctjhoa thanks !

Here's my feedback, I tried this URL: http://localhost:4200/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/89435f76-990b-4f1a-ae6c-c55d96727a23/environment/7826b8eb-76a1-4758-804a-5e62fa86cf32/application/c6b6ca49-b071-4921-99f9-5c3b8f32b322/general

- Source by git repository are not available
Capture d’écran 2023-11-27 à 11 01 54

- It missing the Helm icon and the version badge have a glitch
Capture d’écran 2023-11-27 à 11 02 53

- You can implement setting / source / deployment settings, I don't know if you want do that on this PR ? I see the ticket speak about that

File: libs/shared/ui/src/lib/components/icon/icon.tsx [hunk 2]
```diff
@@ -121,6 +123,10 @@ export function Icon(props: IconProps) {
       return <GenericRegistryIcon {...formattedProps} />
     case IconEnum.GCP_ARTIFACT_REGISTRY:
       return <GcpArtifactRegistryIcon {...formattedProps} />
+    case IconEnum.HELM:
+      return <HelmServiceIcon {...formattedProps} />
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/icon/icon.tsx:127]: Can you have the same name between HelmService and Helm?

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**